cmd/temporal/main.go

```diff
@@ -120,10 +154,20 @@ var commands = map[string]cmd.Cmd{
 							if err != nil {
 								log.Fatal(err)
 							}
-							err = qm.ConsumeMessage("", args["dbPass"], args["dbURL"], args["dbUser"], &cfg)
-							if err != nil {
-								log.Fatal(err)
-							}
+							quitChannel := make(chan os.Signal)
+							signal.Notify(quitChannel, os.Interrupt, syscall.SIGTERM, syscall.SIGINT)
+							ctx, cancel := context.WithCancel(context.Background())
+							waitGroup := &sync.WaitGroup{}
+							waitGroup.Add(1)
+							go func() {
+								if err := qm.ConsumeMessages(ctx, waitGroup, "", args["dbPass"], args["dbURL"], args["dbUser"], &cfg); err != nil {
```

Comment:
hm seeing a few of these in the cmd code, why does this have to run in a goroutine?